Geometry/CMSCommonData/data/materials/2021ZeroMaterial/v1/materials.xml

```diff
@@ -0,0 +1,4621 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
+ <MaterialSection label="materials.xml">
+  <ElementaryMaterial name="Aluminium" density="2.7*g/cm3" symbol=" " atomicWeight="26.98*g/mole" atomicNumber="13"/>
```

Comment:
Please use the Elementary Materials from `Geometry/CMSCommonData/data/materials/2021/v1/materials.xml`. The old list below has errors. I doubt that using correct values for the elements could have much effect on the zero-material studies.

Even better would be to break out the ElementaryMaterial list so it could be used everywhere without copying.

Reply:
So far I was only making the new scenario using older version of materials.xml files. I shall now take them from 2021/v1